cpp/template/cmake/thirdparty/get_raft.cmake

```diff
@@ -46,7 +46,7 @@ function(find_and_configure_raft)
             "BUILD_TESTS OFF"
             "BUILD_PRIMS_BENCH OFF"
             "BUILD_ANN_BENCH OFF"
-            "RAFT_NVTX   ${ENABLE_NVTX}"
+            "RAFT_NVTX ${PKG_ENABLE_NVTX}"
```

Comment:
This doesn't seem to match the variable name `ENABLE_NVTX` on lines 20 and 62.

Reply:
All the parameters of this function are used with `PKG_` before the name. Same goes with `PKG_COMPILE_LIBRARY` used one line below, it is declared without the `PKG_`, but used with `PKG_`

Reply:
Hmm. Maybe that's what
```
cmake_parse_arguments(PKG "${options}" "${oneValueArgs}"
"${multiValueArgs}" ${ARGN} )
```
does.

Sorry, I missed that earlier. This might not work.